### ISSUE TITLE
Write a Vignette

### ISSUE THREAD

**Issue by liwencong1995** [beanumber/nyc311#10]

**Comment by beanumber** [comment]
Add a simple map to the vignette.

**Comment by beanumber** [comment]
The vignette is not bad. I made some light edits. A couple things could make this better:
- more context: include a link to (https://en.wikipedia.org/wiki/3-1-1) to explain what 311 is. Also flesh out the DESCRIPTION. Remember that this is the first thing that anyone will read about this package. 
- The examples in the vignette are not bad, but they could be illuminated a little more. Instead of just picking a random month, is there a specific day that might be interesting? Note this from the Wikipedia: 

```
In New York City, 3-1-1 is used by city officials as one of several sources of measurement and information about the performance of city services. Important dates in the history of New York's 3-1-1 service include December 20, 2005, when it received its record high of 240,000 calls, due to the first day of the 2005 New York City transit strike, and June 20, 2007, when it received its 50 millionth call.[2]
```

Consider this as a mini-DataFest-style exercise for these data. What interesting nuggets can you find? It's worth having a Google to see if there are people who have already done anything interesting with these data.

**Comment by liwencong1995** [comment]
We can only get access to phone call information from 2010-01-10, so I will take a look at the data collected on St. Patrick’s Day, 2014.

**Comment by liwencong1995** [comment]
Should I include a list of variables in the vignette? There are more than 30 variables in this dataset; should I only include a subset of the variables?

**Comment by beanumber** [comment]
I wasn't getting the same results on the St. Patrick's Day data. Also, these take too long to download. The vignette should compile in a matter of seconds. Can we figure out a way around this? What about using New Year's Day instead, and then limiting the number of calls?

**Comment by liwencong1995** [comment]
If I take a subset of calls, I won’t be able to tell what is the the most frequently happened issue type. But I can figure something else? I can create a map for a subset of issues happened on St. Patrick's Day.

I made some changes to the vignette yesterday. I took a subset of 100 calls and mapped them on a map of New York State. Since the St. Patrick's Day Parade happened in New York City, I expect people report about issues happened in Manhattan.
